tests/integration/cloudformation/test_cloudformation_route53.py

```diff
@@ -0,0 +1,80 @@
+from localstack.utils.strings import short_uid
+from localstack.utils.sync import wait_until
+from tests.integration.cloudformation.utils import load_template_raw
+
+
+def test_create_record_set_via_id(
+    route53_client,
+    cfn_client,
+    is_change_set_created_and_available,
+    cleanup_changesets,
+    cleanup_stacks,
+    is_stack_created,
+):
+    stack_name = f"stack-{short_uid()}"
+    change_set_name = f"change-set-{short_uid()}"
+    route53_name = f"www.{short_uid()}.com."
+    caller_ref = f"caller-ref-{short_uid()}"
+    create_zone_response = route53_client.create_hosted_zone(
+        Name=route53_name, CallerReference=caller_ref
+    )
+    hosted_zone_id = create_zone_response["HostedZone"]["Id"]
+
+    template_rendered = load_template_raw("route53_hostedzoneid_template.json")
+    template_rendered = template_rendered % (hosted_zone_id, route53_name)
+    response = cfn_client.create_change_set(
+        StackName=stack_name,
+        ChangeSetName=change_set_name,
+        TemplateBody=template_rendered,
+        ChangeSetType="CREATE",
+    )
+
+    change_set_id = response["Id"]
+    stack_id = response["StackId"]
+    try:
+        wait_until(is_change_set_created_and_available(change_set_id))
+        cfn_client.execute_change_set(ChangeSetName=change_set_id)
+        wait_until(is_stack_created(stack_id))
+        describe_result = cfn_client.describe_stacks(StackName=stack_id)
+        assert describe_result["Stacks"][0]["StackStatus"] == "CREATE_COMPLETE"
+
+    finally:
+        cleanup_changesets([change_set_id])
+        cleanup_stacks([stack_id])
```

Comment:
```suggestion
    deploy_cfn_template(template_filename="route53_hostedzoneid_template.json")
```

could be replaced with the usage of the cfn fixture

Reply:
alright, I think I simply took a look on some existing test to copy the behaviour, but will do ofc :+1: .